indexer/src/main/java/au/org/aodn/esindexer/configuration/GeoNetworkSearchConfig.java

```diff
@@ -17,16 +18,37 @@
 import org.springframework.http.MediaType;
 import org.springframework.web.client.RestTemplate;
 
+import java.lang.reflect.Field;
+import java.util.Set;
+
 @Configuration
 public class GeoNetworkSearchConfig {
 
     @Bean(name = "gn4ElasticsearchClient")
-    public ElasticsearchClient createGN4ElasticsearchClient(@Qualifier("gn4ElasticRestClient") RestClient restClient) {
+    public ElasticsearchClient createGN4ElasticsearchClient(@Qualifier("gn4ElasticRestClient") RestClient restClient) throws NoSuchFieldException, IllegalAccessException {
 
         RestClientTransport c = new RestClientTransport(restClient, new JacksonJsonpMapper());
+        // Fix due to elastic search api update
+        // Need a hack to work around an issue with geonetwork, geonetwork exposed the ElasticSearch api via endpoint _search
```

Comment:
can you add a TODO for easier to track in future?

Reply:
if there will be something to do later for getting away from work around strategy